### PR TITLE
Restores Security Culture

### PR DESCRIPTION
**Pull request by SconesC**
STOP OR I'LL BASH YOU

Not sure why these were removed in the first place. I was tempted to put SWAT masks back, but these will have to do. Warden can keep his spares.

### CODE REVIEW

**Comment by WJohn** [comment]
They were allegedly removed by paprika for being too spam-y. Whether they're that annoying or not can be left up to the players, I'm fine with or without them being so openly available.

**Comment by SconesC** [comment]
'Tis an ancient Security rite to converse entirely between one another using these things.

**Comment by Fox-McCloud** [comment]
STOP BREAKING THE LAW ASSHOLE!

File: code/game/objects/structures/crates_lockers/closets/secure/security.dm [hunk 1]
```diff
@@ -113,6 +113,7 @@
 	new /obj/item/weapon/grenade/flashbang(src)
 	new /obj/item/weapon/storage/belt/security/full(src)
 	new /obj/item/clothing/glasses/hud/security/sunglasses(src)
+	new /obj/item/clothing/mask/gas/sechailer
```
Comment (Iamgoofball) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: forgot to do (src) at the end

Comment (SconesC) [inline code/game/objects/structures/crates_lockers/closets/secure/security.dm]: thanks fam

**Comment by pudl** [comment]
The ugly sprite Paprka gave them (that makes them look like the Bane meme) should be replaces with the old sprite. I don't understand why Paprka got mad about sec gas masks making people faceless when normal gas masks do the exact same thing.

**Comment by SconesC** [comment]
I'll make a poll to see if people want the SWAT masks

**Comment by GunHog** [comment]
I personally prefer the SWAT masks, and yes give the officers their hailers back! :+1:

**Comment by Steelpoint** [comment]
Here's what happened.

Paprika really disliked the old SWAT style Security Masks because he felt it contributed heavily to the 'shitcurity' culture of security and he really disliked the concept of SS13 security looking as they do now

(I should stress that for the longest time Paprika was working to reel back Security's appearance to look akin to mall cops, at one point he did put up a PR that did just that)

So he gutted the old Sec masks and refused to budge.

Furthermore he also took the masks out of the Sec lockers and hid them in the Armoury. So the only way you could get a Sec Mask was if you asked the Warden or the Head of Security.

---

Personally I think the Sec Masks were always fine and would support them being readded :+1:

**Comment by Fox-McCloud** [comment]
The only thing I really like about the current ones is that you can fold them down and still use the hail feature.

**Comment by Iamgoofball** [comment]
@Steelpoint what no

I did the mall cops pr nerd, lemme go find it

**Comment by Steelpoint** [comment]
Pap also initially proposed a mall cops sprite set up in one of the numerous sec sprites PRs.

He then swapped that out to the corporate sec sprites.

**Comment by pudl** [comment]
@Steelpoint The PR before Paprka's corporate security was not mallcop sec, instead it simply changed the color of oldsec (Ausop's sprite) to match the rest of security. People were upset because they thought that a brighter shade of red made security more approachable, or something like that.

**Comment by Cheridan** [comment]
These things are so fucking annoying, nobody even uses them except to spam on the escape shuttle.

**Comment by freerealestate** [comment]
>These things are so fucking annoying

Yeah, this.

**Comment by MrPerson** [comment]
Making security faceless is bad. We've been over this before in great detail and I'd rather not rehash it.

**Comment by Steelpoint** [comment]
Then get rid of the talking command.

**Comment by Incomptinence** [comment]
Did making security have an  exposed face really improve things? 

I missed any point where parties in favour of it were not treating security the same way with or without.

**Comment by SconesC** [comment]
@Cheridan 

So is that an executive no, or just you voicing displeasure?

@MrPerson 

Security having their faces exposed has changed pretty much nothing. Interactions between them and the crew is the same as always; masks have not been any sort of factor.

**Comment by Iamgoofball** [comment]
@cheridan I use them all the time, and when they were in I saw people using them as well, for legitimate not spamming purposes.

I dunno what shitsec you were playing with, but the security officers I was seeing use them actually used them for their purpose: yell at people and alert them to their arrest in a split-second, instead of giving them 20 seconds to beat you to death while you're typing

**Comment by Saltycut** [comment]
:+1: Bring them back

**Comment by SconesC** [comment]
@Cheridan 
Past the time limit, waiting on the chief funnymen to merge

if these prove problematic I'll fix it

**Comment by SconesC** [comment]
oh well

**Comment by Thunder12345** [comment]
Nooo! The culture!

**Comment by pudl** [comment]
@Thunder12345 I'm doing it in a bigger PR.

**Pull request closed without merging**